mindsdb/integrations/handlers/youtube_handler/youtube_tables.py

```diff
@@ -5,12 +5,10 @@
 from mindsdb.utilities.log import get_log
 
 from mindsdb_sql.parser import ast
+from mindsdb.integrations.handlers.utilities.query_utilities import SELECTQueryParser, SELECTQueryExecutor
 
-
-from googleapiclient.discovery import build
-from googleapiclient.errors import HttpError
```

Comment:
It looks like these imports are not used in any of the table implementations. Thanks for cleaning it up. I think there is a similar import that is not required in the handler module, let's remove that as well, just to be clean.

Reply:
I see that you've already done this in the other PR. Thanks!